lib/index.js

```diff
@@ -556,7 +556,7 @@ Config.prototype = {
         var ycb;
 
         if (dimensions) {
-            ycb = makeYCB(this, dimensions, contents);
+            ycb = makeYCB(this, dimensions, contents, path);
         } else {
             ycb = makeFakeYCB(dimensions, contents);
```

Comment:
Should `makeFakeYCB` match?

Reply:
`makeFakeYCB` creates a simple dummy object so there's no logging/errors when called. The path param would be unused.